packages/swapper/src/constants.ts

```diff
@@ -128,8 +127,17 @@ export const getDefaultSlippageDecimalPercentageForSwapper = (
     case SwapperName.Chainflip:
       return DEFAULT_CHAINFLIP_SLIPPAGE_DECIMAL_PERCENTAGE
     case SwapperName.Jupiter:
-      return DEFAULT_JUPITER_SLIPPAGE_DECIMAL_PERCENTAGE
+      throw new Error('Default slippage not supported by Jupiter')
     default:
-      assertUnreachable(swapperName)
+      return assertUnreachable(swapperName)
```

Comment:
tsc wasn't very happy without actually returning (already the case in develop) though not actually throwing `tsc` compile errors